src/audio/audio.cpp

```diff
@@ -36,6 +37,37 @@
 #endif
 
 /**
+@internal
+
+@class Audio::Private
+
+@brief Encapsulates private audio framework from public qTox Audio API.
+*/
+class Audio::Private
+{
+public:
+    Private()
+        : minInputGain{-30.0}
+        , maxInputGain{30.0}
+        , gain{0.0}
+        , gainDecibel{0.0}
+    {
+    }
+
+    void setInputGain(qreal dB)
+    {
+        gainDecibel = qBound(minInputGain, dB, maxInputGain);
+        gain = qPow(10.0, (dB * 0.05));
+    }
+
+public:
```

Comment:
Keep these fields public somewhat defeats the above function. Any function can arbitarily modify the fields without the logic defined above. In particular, it allows for the consistentcy of the four parameters to be broken by mistakes that are no easily seen (since they can be anywhere in the code base that uses this code). I understand that mutation is required by the `Audio` class, see my line comments below.